src/libs/UserUtils.ts

```diff
@@ -83,21 +100,39 @@ function getDefaultAvatar(accountID = -1): React.FC<SvgProps> {
 
     // There are 24 possible default avatars, so we choose which one this user has based
     // on a simple modulo operation of their login number. Note that Avatar count starts at 1.
-    const accountIDHashBucket = ((accountID % CONST.DEFAULT_AVATAR_COUNT) + 1) as AvatarRange;
-
+    let accountIDHashBucket: AvatarRange;
+    if (avatarURL && /images\/avatars\/default-avatar_\d+\./.test(avatarURL)) {
```

Comment:
Let's also add an explanatory comment here as it won't be super clear why we're extracting the ID from `avatarURL`.

Reply:
Comment added